indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -59,6 +56,18 @@
             });
         });
 
+        $bulkMoveButton.on('click', function(evt) {
+            evt.preventDefault();
+            if ($(this).hasClass('disabled')) {
+                return;
+            }
+            _moveCategories(getSelectedCategories(), _categories[categoryId], $(this).data('href'));
+        });
+
+        $table.find('.js-move-category').on('click', function(evt) {
+            _moveCategories([$(this).data('categoryId')], _categories[categoryId], $(this).data('href'));
```

Comment:
Not a strong opinion, so I'll go for that then. In any case, I found this performance comparison: http://stackoverflow.com/questions/10433014/what-is-the-cost-of-this